app/resources/api/v2/request_resource.rb

```diff
@@ -23,6 +23,7 @@ class RequestResource < BaseResource
       has_one :request_type, always_include_linkage_data: true
       has_one :primer_panel
       has_one :pre_capture_pool
+      has_many :poly_metadata, as: :metadatable, class_name: 'PolyMetadatum', dependent: :destroy
```

Comment:
What does it do, putting the `dependent: destroy` here in the API resource, rather than in the model?

Reply:
Possibly (probably) nothing, have removed it from here and added it to model.